solr/core/src/java/org/apache/solr/search/facet/FacetRequest.java

```diff
@@ -234,9 +234,8 @@ public Query createDomainQuery(FacetContext fcontext) throws IOException {
         // the number of classes that have to know about the number of possible settings on the join
         // (ie: if we add a score mode, or some other modifier to how the joins are done)
 
-        final SolrConstantScoreQuery fromQuery = new SolrConstantScoreQuery(fcontext.base.makeQuery());
+        final ConstantScoreQuery fromQuery = new ConstantScoreQuery(fcontext.base.makeQuery());
         // this shouldn't matter once we're wrapped in a join query, but just in case it ever does...
-        fromQuery.setCache(false);
```

Comment:
Why did this go away?

Reply:
I couldn't see how it was necessary. Why would we still need it?

Reply:
The intent here looks to be that we don't want to cache the individual sub-queries that would get wrapped in the JoinQuery later (presumably because we are caching the whole join?)

Since lucene query doesn't have this idea of caching, we need to wrap fromQuery in an ExtendedQuery to disable the caching.

Reply:
Thanks for the explanation! I found it really helpful